beacon-chain/sync/initial-sync/blocks_fetcher.go

```diff
@@ -571,6 +637,58 @@ func (f *blocksFetcher) fetchBlobsFromPeer(ctx context.Context, bwb []blocks2.Bl
 	return nil, errNoPeersAvailable
 }
 
+// fetchColumnsFromPeer fetches blocks from a single randomly selected peer.
+func (f *blocksFetcher) fetchColumnsFromPeer(ctx context.Context, bwb []blocks2.BlockWithROBlobs, pid peer.ID, peers []peer.ID) ([]blocks2.BlockWithROBlobs, error) {
+	ctx, span := trace.StartSpan(ctx, "initialsync.fetchColumnsFromPeer")
+	defer span.End()
+	if slots.ToEpoch(f.clock.CurrentSlot()) < params.BeaconConfig().DenebForkEpoch {
```

Comment:
`f.clock.CurrentSlot()` is called several times in the function.
It can be factorized in a `currentSlot` variable.